core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -95,7 +95,8 @@ class LoadBalancerService(
     info(this, s"blackboxFraction = $blackboxFraction")
 
     /** We run this often on an invoker before going onto the next. */
-    private val activationCountBeforeNextInvoker = 10
+    private val activationCountBeforeNextInvoker = Math.max(1, config.loadbalancerActivationCountBeforeNextInvoker.toInt)
```

Comment:
yup that was my thinking as well - there are some more of these `toInts` that we should get rid of too elsewhere (noting here for the record: rate throttler and sequence limit come to mind).